r/R/dataset.R

```diff
@@ -64,6 +67,8 @@ open_dataset <- function(sources,
                          partitioning = hive_partition(),
                          unify_schemas = NULL,
                          ...) {
+  # TODO: default to unify_schemas = TRUE when `sources` is a vector of file paths?
+  # (if so, also change the unifiy_schemas param description in the docs entry above)
```

Comment:
Should `unify_schemas` default to `TRUE` when `sources` is a vector of file paths (not a directory path)?